apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/attach-launch-sidebar.tsx

```diff
@@ -41,9 +42,11 @@ registerPlugin( 'a8c-editor-site-launch', {
 		}
 
 		return (
-			<LaunchContext.Provider value={ { siteId: window._currentSiteId } }>
-				<LaunchModal onClose={ closeSidebar } />
-			</LaunchContext.Provider>
+			<I18nProvider>
```

Comment:
You don't even need to mount the provider here: the default context value, derived from `undefined` `localeData`, will be used in that case.

Reply:
Good thinking, I'd somehow missed this in the midst of all these changes. I actually like this behaviour 😎